Dockerfile

```diff
@@ -1,23 +1,17 @@
-# syntax=docker/dockerfile:1
-
-FROM golang:1.21
-
-WORKDIR /wotlk
-COPY . .
-COPY gitconfig /etc/gitconfig
-
-RUN apt-get update
-RUN apt-get install -y protobuf-compiler
-RUN go get -u google.golang.org/protobuf
-RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
-
-RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
-
-ENV NODE_VERSION=19.8.0
-ENV NVM_DIR="/root/.nvm"
-RUN . "$NVM_DIR/nvm.sh" && nvm install ${NODE_VERSION}
-RUN . "$NVM_DIR/nvm.sh" && nvm use v${NODE_VERSION}
-RUN . "$NVM_DIR/nvm.sh" && nvm alias default v${NODE_VERSION}
-ENV PATH="/root/.nvm/versions/node/v${NODE_VERSION}/bin/:${PATH}"
-
-EXPOSE 8080/tcp
\ No newline at end of file
+# syntax=docker/dockerfile:1
+FROM golang:1.21
+
+WORKDIR /wotlk
+COPY . .
+COPY gitconfig /etc/gitconfig
+
+RUN apt-get update
+RUN apt-get install -y protobuf-compiler
+RUN go get -u google.golang.org/protobuf
+RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+
+ENV NODE_VERSION=18.18.0
```

Comment:
I see this is downgraded from 19.X to 18.X. Was there a reason you'd need to do that?

Reply:
No, not that I know off. I think this might be a faulty merge.